kernel/siw_cm.h

```diff
@@ -94,6 +99,9 @@ struct siw_cep {
 	struct siw_mpa_info	mpa;
 	int			ord;
 	int			ird;
+	int			revision;
+	int			p2ptype;
+	int			enhanced_rdma_connection;
```

Comment:
Changed to enhanced_rdma_conn_est. I put them into siw_cep since that is where ord/ird is located. But I can move them if you think that's best.